### DISCUSSION TITLE
能不能弄一个视频暂停不要显示封面的开关？

### DISCUSSION THREAD

**Discussion by doio** [the1812/Bilibili-Evolved#3742] [功能建议 / Ideas]
现在 bilibili 的播放器好像有改动，视频播放暂停的时候会显示封面图片，可是如果要截图的话就截图不到暂停时候的画面了。

我看到有「播放前显示封面」的功能，能不能添加一个「播放暂停时不显示封面」的开关呢？

**Comment by the1812** [comment]
本来就不应该显示 (#3698)
